db/schema.sql

```diff
@@ -0,0 +1,19 @@
+DROP TABLE IF EXISTS tenants;
+DROP TABLE IF EXISTS apartments;
+
+CREATE TABLE tenants (
+  id SERIAL PRIMARY KEY,
+  name TEXT,
+  age INTEGER,
+  gender TEXT,
+  apartment_id INTEGER
+);
+
+CREATE TABLE apartments (
+  id SERIAL PRIMARY KEY,
+  address TEXT,
+  monthly_rent INTEGER,
+  sqft INTEGER,
+  num_beds INTEGER,
+  num_baths INTEGER
+);
```

Comment:
Your `schema.sql` file looks fine to me. If you run `psql -d landlord_db < db/schema.sql`, it should set up your tables properly. You just need to make sure that you're adding data to your database as well, whether that's through a seed file or interacting with your database using `binding.pry`.